cluster-autoscaler/core/scaledown/planner/planner.go

```diff
@@ -289,14 +292,15 @@ func (p *Planner) categorizeNodes(podDestinations map[string]bool, scaleDownCand
 		// TODO(x13n): break on timeout. Figure out how to handle nodes
 		// identified as unneeded in previous iteration, but now
 		// skipped due to timeout.
-		removable, unremovable := p.rs.SimulateNodeRemoval(node, podDestinations, p.latestUpdate, pdbs)
+		removable, unremovable := p.rs.SimulateNodeRemoval(node, podDestinations, p.latestUpdate, p.pdbRemainingDisruptions.GetPdbs())
```

Comment:
This cannot rely on PDB checking inside `SimulateNodeRemoval`, `CanDisrupt` should be explicitly checked.